Commons/DataTypes.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace TourManager.CommonTypes
```

Comment:
My approach is that the following fields should never be none in the record. Otherwise the whole logic breaks. That's why those which are required I haven't put value "none". For empty records they'll be null I guess.